ansible/roles/master/tasks/download_bins.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Create bin dir
+  file: path={{ bin_dir }} state=directory
+
+- name: Create directory for current Kubernetes release
+  file: path={{ kube_current_release_directory }} state=directory
+
+- name: Set a list of kube master binaries to download
+  set_fact:
+    kube_binaries: "{{ kube_master_binaries }}"
+
+- name: Extend the list of kube master binaries with kubelet
+  set_fact:
+    kube_binaries: "{{ kube_master_binaries }} kubelet"
+  when: networking == 'opencontrail'
+
+- name: Download Kubernetes binaries
+  get_url:
+    url: "{{ kube_download_url_base }}/{{ item }}"
+    dest: "{{ kube_current_release_directory }}/{{ item }}"
+    mode: 0755
+    validate_certs: False
```

Comment:
As currently all the binaries are downloaded directly from github.com/kubernetes/kubernetes there is no need for it. But yes, is should be validated by default.